README.md

```diff
@@ -44,6 +44,26 @@ Example command to spin up 2 Windows nodes and retain them after test run:
 hack/run-ci-e2e-test.sh -s -k "openshift-dev" -n 2      
 ```
 
+## Running the operator locally
+To run the operator locally (for testing purpose only).
+We need to set up all the environment variables required in [Testing Locally](#testing-locally) as well as 
```

Comment:
`Testing locally` should also be changed.

Reply:
Folks can misunderstand `local` cluster  to mean a cluster on their laptop. Similarly `local` testing can imply that the operator is running on their laptop. With WMCO we can no longer run locally but always on the remote cluster. So this needs to be restructured as some sort of development workflow.